node/parachain/src/service.rs

```diff
@@ -56,7 +78,10 @@ pub fn new_partial(
 	>,
 	sc_service::Error,
 > {
-	let inherent_data_providers = sp_inherents::InherentDataProviders::new();
+	// TODO this author id should not be hard coded. This is just for simple testing. We need to
+	// wire this back to the CLI.
```

Comment:
@JoshOrndorff why not just pass in `author: H160` as an input to this `new_partial` function like before?